bio/goleft/indexcov/wrapper.py

```diff
@@ -0,0 +1,55 @@
+# coding: utf-8
+
+__author__ = "Thibault Dayris"
+__mail__ = "thibault.dayris@gustaveroussy.fr"
+__copyright__ = "Copyright 2024, Thibault Dayris"
+__license__ = "MIT"
+
+from tempfile import TemporaryDirectory
+from snakemake import shell
+
+extra = snakemake.params.get("extra", "")
+log = snakemake.log_fmt_shell(stdout=True, stderr=True, append=True)
+
+with TemporaryDirectory() as tempdir:
+    shell(
+        "goleft indexcov "
+        "{extra} "
+        "--directory {tempdir}/out "
+        "--fai {snakemake.input.fai} "
+        "{snakemake.input.aln} "
+        "{log} "
+    )
+
+    if "ped" in snakemake.output.keys():
```

Comment:
Some wrappers use the form:
```suggestion
    if snakemake.output.get("ped"):
```

not sure if it makes a difference.

Reply:
I test existence of the key in an object. The suggestion tries to get a key from an object and, it it fails, returns `None` object, then evaluates `None` as `False` or not-`None` as `True`.

In terms of time it's almost alike:

``` python
In [1]: def foo(d):
   ...:     if d.get("key"):
   ...:         return "Case1"
   ...:     return "Case2"
   ...: 

In [2]: def bar(d):
   ...:     if "key" in d:
   ...:         return "Case1"
   ...:     return "Case2"
   ...: 

In [3]: d = {"key": "exists"}

In [4]: %timeit foo(d)
92.6 ns ± 1.4 ns per loop (mean ± std. dev. of 7 runs, 10,000,000 loops each)

In [5]: %timeit bar(d)
76.2 ns ± 0.12 ns per loop (mean ± std. dev. of 7 runs, 10,000,000 loops each)

In [6]: e = {"not_key": "does not exists"}

In [7]: %timeit foo(e)
85.7 ns ± 0.128 ns per loop (mean ± std. dev. of 7 runs, 10,000,000 loops each)

In [8]: %timeit bar(e)
73.1 ns ± 0.148 ns per loop (mean ± std. dev. of 7 runs, 10,000,000 loops each)
```

Time difference won't make any difference even if my solution is faster. If you see a difference I did not mention, please tell me.

Do you want me to make the changes ?